core/src/main/java/com/digitalpebble/stormcrawler/persistence/urlbuffer/SchedulingURLBuffer.java

```diff
@@ -27,8 +27,8 @@
 import java.util.Queue;
 import java.util.concurrent.TimeUnit;
 import org.apache.storm.tuple.Values;
-import org.checkerframework.checker.nullness.qual.NonNull;
```

Comment:
Do we know, who brings `checkerframework` in the project? Is it even reqired, after adding jetbrain annotations?

Reply:
That was an oversight by me, I'll get to it.

Reply:
Looking at the tree it is from caffeine:
```
[INFO] +- com.github.ben-manes.caffeine:caffeine:jar:2.9.2:compile
[INFO] |  +- org.checkerframework:checker-qual:jar:3.10.0:compile
[INFO] |  \- com.google.errorprone:error_prone_annotations:jar:2.5.1:compile
```
But I'm not that accustomed to maven and pom.xml, what is the proper way of removing it? Just excluding like this?

```xml
<dependency>
	<groupId>com.github.ben-manes.caffeine</groupId>
	<artifactId>caffeine</artifactId>
	<version>${caffeine.version}</version>
	<exclusions>
		<exclusion>  <!-- declare the exclusion here -->
			<groupId>org.checkerframework</groupId>
			<artifactId>checker-qual</artifactId>
		</exclusion>
	</exclusions>
</dependency>
```

Reply:
if it is needed by Caffeine, don't exclude it. Just let it be